src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/DoNotDecreaseInheritedMemberVisibility.cs

```diff
@@ -1,86 +0,0 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System.Collections.Immutable;
-using Analyzer.Utilities;
-using Microsoft.CodeAnalysis.Diagnostics;
-using Microsoft.CodeAnalysis;
-using System.Linq;
-using Analyzer.Utilities.Extensions;
-
-namespace Microsoft.CodeQuality.Analyzers.ApiDesignGuidelines
-{
-    /// <summary>
-    /// CA2222: Do not decrease inherited member visibility
-    /// </summary>
-    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
-    public class DoNotDecreaseInheritedMemberVisibilityAnalyzer : DiagnosticAnalyzer
-    {
-        internal const string RuleId = "CA2222";
-
-        private static readonly LocalizableString s_localizableTitle = new LocalizableResourceString(nameof(MicrosoftApiDesignGuidelinesAnalyzersResources.DoNotDecreaseInheritedMemberVisibilityTitle), MicrosoftApiDesignGuidelinesAnalyzersResources.ResourceManager, typeof(MicrosoftApiDesignGuidelinesAnalyzersResources));
-
-        private static readonly LocalizableString s_localizableMessage = new LocalizableResourceString(nameof(MicrosoftApiDesignGuidelinesAnalyzersResources.DoNotDecreaseInheritedMemberVisibilityMessage), MicrosoftApiDesignGuidelinesAnalyzersResources.ResourceManager, typeof(MicrosoftApiDesignGuidelinesAnalyzersResources));
-        private static readonly LocalizableString s_localizableDescription = new LocalizableResourceString(nameof(MicrosoftApiDesignGuidelinesAnalyzersResources.DoNotDecreaseInheritedMemberVisibilityDescription), MicrosoftApiDesignGuidelinesAnalyzersResources.ResourceManager, typeof(MicrosoftApiDesignGuidelinesAnalyzersResources));
```

Comment:
❗️ The resource strings used only by this analyzer and/or code fix should be removed as well.

Reply:
@sharwell fixed.